admin/tests-integration/check-form.service.spec.js

```diff
@@ -18,7 +18,7 @@ describe('check-form.service', () => {
   const availableForms = []
   const seenForms = []
 
-  describe('in-memory tests of check-form.service.allocateCheckForm', () => {
+  xdescribe('in-memory tests of check-form.service.allocateCheckForm', () => {
```

Comment:
how come this is disabled?

Reply:
1. - it only generates a csv - it does do the analysis in the test
2. - it takes about 2 minutes to run as it runs 600k simulations